README.md

```diff
@@ -21,6 +21,7 @@ Caption Studio is designed to help you in creating audio caption files (JSON for
 
 ## Project setup
 ```
+nvm use
 npm install
```

Comment:
probably worth adding an alternative comment for the asdf usage just to be thorough

Reply:
Just to clarify would the command be `asdf install nodejs`

Reply:
I think so, might just be `asdf install` since we're only doing the one tool version anyway? But the docs aren't super comprehensive so I'll follow your lead